src/Juvix/Compiler/Core/Translation/FromInternal.hs

```diff
@@ -423,6 +425,20 @@ goAxiomDef a = case a ^. Internal.axiomBuiltin >>= builtinBody of
           )
       Internal.BuiltinString -> Nothing
       Internal.BuiltinIO -> Nothing
+      Internal.BuiltinTrace ->
+        Just
+          ( mkLambda'
+              ( mkLambda'
+                  ( mkLambda'
+                      ( mkLambda'
+                          ( mkBuiltinApp' OpTrace [mkVar' 1, mkVar' 0]
+                          )
+                      )
+                  )
+              )
+          )
+      Internal.BuiltinFail ->
+        Just (mkLambda' (mkLambda' (mkBuiltinApp' OpFail [mkVar' 0])))
```

Comment:
This is wrong: `fail` takes one argument not two.

Reply:
Why not: {a:U} -> String -> a?

Reply:
Sorry, OK, this is for type arguments. Makes sense